addons/web/static/src/legacy/js/views/list/list_renderer.js

```diff
@@ -176,8 +176,11 @@ var ListRenderer = BasicRenderer.extend({
                 aggregateValue = Infinity;
             }
             _.each(data, function (d) {
-                count += 1;
                 var value = (d.type === 'record') ? d.data[attrs.name] : d.aggregateValues[attrs.name];
+                if (Number(value) !== value) {
```

Comment:
@adwid After reading your commit message, could it be just `if (!value)` or is your comparaison with `Number()` is necessary ?

Reply:
With this condition, we would ignore the zeros while the zeros could be a relevant information